css/style.css

```diff
@@ -0,0 +1,147 @@
+/* style.css */
+
+* {
+  box-sizing: border-box;
+}
+
+.clearfix:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+
+.hero {
+  background-image: url(../images/mountain2.jpg);
+  background-size: cover;
+  color: white;
+  height: calc(100vh - 50px);
+
+
+
+/* Hey future Emileigh: This is flex box. it lets you center things and is amazing!*/
```

Comment:
:+1: future Emileigh will probably thank you and be amazing at your wisdom and foresight